### PR TITLE
Clean properties

### PR DESCRIPTION
**Pull request by EpokK**
Handle group in vCard properties
Handle pref in vCard properties (email)

### CODE REVIEW

File: src/app/helpers/properties.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    return properties.filter(({ value }) => value);
+};
+
+/**
+ * Add `pref` to email, adr, tel to save order
+ * @param {Array} properties
+ * @param {Array}
+ */
+export const addPref = (properties = []) => {
+    const prefs = { email: 0, tel: 0, adr: 0 };
```
Comment (econdepe) [inline src/app/helpers/properties.js]: Could we start to count in `1`?
Quoting the vCard RFC:

>   5.3.  PREF
> 
>    The PREF parameter is OPTIONAL and is used to indicate that the
>    corresponding instance of a property is preferred by the vCard
>    author.  Its value MUST be an integer between 1 and 100 that
>    quantifies the level of preference.  Lower values correspond to a
>    higher level of preference, with 1 being most preferred.

Also in CSV files the headers are always "Email address/Phone Number/Address 1", "Email address/Phone Number/Address 2", etc, which would make more natural to start the count in 1.

Comment (EpokK) [reply src/app/helpers/properties.js]: We are:
```js
const pref = (prefs[property.field] = prefs[property.field] + 1);

        return {
            ...property,
            pref
        };
```

Comment (econdepe) [reply src/app/helpers/properties.js]: oh, I misunderstood. But then this relies on the `properties` to be ordered by preference already. If I apply `addPref` to 
```
[{ field: 'email', pref: 2, value: 'notFavorite'}, { field: 'email', pref: 1, value: 'favorite'}]
```
I'll get
```
[{ field: 'email', pref: 1, value: 'notFavorite'}, { field: 'email', pref: 2, value: 'favorite'}]
```

Comment (EpokK) [reply src/app/helpers/properties.js]: Yeah, [we order pref when we build the contact properties](https://github.com/ProtonMail/proton-contacts/blob/master/src/app/helpers/vcard.js#L110).

**Pull request merged**